### DISCUSSION TITLE
how to blocklist everything for a specific client?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3497] [Q&A]
I used to block everything on pihole using *. regex can i do that here too?

block everything for a specific client?

thanks

**Comment by ameshkov** [comment]
Something like that would work just okay: `$client=ClientName`

Please check out the doc on the filtering rules syntax to learn more about that: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists
